### DISCUSSION TITLE
Soundswitch is crashing after a few hours.

### DISCUSSION THREAD

**Discussion by sagarbattin** [Belphemur/SoundSwitch#1071] [Q&A]
Soundswitch is crashing after a few hours.  I see below in the event viewer.  Can anyone please help me on this?

Faulting application name: SoundSwitch.exe, version: 6.4.2.0, time stamp: 0x62b2fac5
Faulting module name: ntdll.dll, version: 10.0.22000.282, time stamp: 0xb594bedc
Exception code: 0xc0000374
Fault offset: 0x000000000010be99
Faulting process id: 0x3f68
Faulting application start time: 0x01d90e0b1eb52063
Faulting application path: C:\Program Files\SoundSwitch\SoundSwitch.exe
Faulting module path: C:\Windows\SYSTEM32\ntdll.dll
Report Id: ad48f8e5-f1a1-42c4-b4cf-9813425f0f3b
Faulting package full name: 
Faulting package-relative application ID:

**Comment by lambdakilo** [comment]
Seems to be a Windows issue. Did updating your Windows fix it for you?
